libdnf5-cli/output/advisorylist.cpp

```diff
@@ -104,18 +107,68 @@ void print_advisorylist_table(
     scols_unref_table(table);
 }
 
+static json_object * adv_pkg_as_json(auto & adv_pkg) {
+    auto advisory = adv_pkg->get_advisory();
+    json_object * json_advisory = json_object_new_object();
+    json_object_object_add(json_advisory, "name", json_object_new_string(advisory->get_name().c_str()));
+    json_object_object_add(json_advisory, "type", json_object_new_string(advisory->get_type().c_str()));
+    json_object_object_add(json_advisory, "severity", json_object_new_string(advisory->get_severity().c_str()));
+    json_object_object_add(json_advisory, "nevra", json_object_new_string(adv_pkg->get_nevra().c_str()));
+    unsigned long long buildtime = advisory->get_buildtime();
+    json_object_object_add(
+        json_advisory, "buildtime", json_object_new_string(libdnf5::utils::string::format_epoch(buildtime).c_str()));
+    return json_advisory;
+}
+
+static json_object * adv_refs_as_json(
+    auto & pretty_reference_type, auto & reference_type, auto & adv_pkg, auto & advisory) {
+    json_object * json_advisory = json_object_new_object();
+    json_object_object_add(json_advisory, "advisory_id", json_object_new_string(advisory.get_name().c_str()));
```

Comment:
What do you think about calling this `name` instead of `id`?
Just to be consistent with the output without references and with the API.

Reply:
In the `updateinfo.xml` it is referred to as `id`:
https://github.com/rpm-software-management/dnf5/blob/65076dc83ecccf3d75d18271c8922a7ab3c0cef1/doc/repository_format/updateinfo-permissive.xsd#L86-L105

So I guess it's a bit of a preference here on if we continue with the `name` as is in the table output, or use `id` that is in the source data.

I don't have a strong opinion here, happy to use `name`